src/WebJobs.Script/Binding/Http/HttpBinding.cs

```diff
@@ -120,6 +122,11 @@ internal static void ParseResponseObject(IDictionary<string, object> responseObj
                 {
                     enableContentNegotiation = enableContentNegotiationValue;
                 }
+
+                if (responseObject.TryGetValue("cookies", out List<Tuple<string, string, CookieOptions>> cookiesValue, ignoreCase: true))
```

Comment:
I see we are using strings `cookies`, `body` etc through out this file. We should move them separate file `Httpconstants` file. It is out-of-scope for this PR. Please open a different issue.